### ISSUE TITLE
Failed to resolve OS dependant maven artifacts

### ISSUE THREAD

**Issue by galderz** [maxandersen/jbang#48]
With version `0.13.2`. Was working with `0.8.1`:

```
jbang docker_java.java
[jbang] Resolving dependencies...
[jbang]     Resolving com.github.docker-java:docker-java:3.1.5...Feb 03, 2020 3:18:58 PM org.jboss.shrinkwrap.resolver.impl.maven.logging.LogTransferListener transferFailed
WARNING: Failed downloading io/netty/netty-transport-native-unix-common/4.1.31.Final/netty-transport-native-unix-common-4.1.31.Final-$%7Bos.detected.name%7D-$%7Bos.detected.arch%7D.jar from https://jcenter.bintray.com/. Reason:
org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.netty:netty-transport-native-unix-common:jar:${os.detected.name}-${os.detected.arch}:4.1.31.Final in jcenter (https://jcenter.bintray.com/)
Feb 03, 2020 3:18:59 PM org.jboss.shrinkwrap.resolver.impl.maven.logging.LogTransferListener transferFailed
WARNING: Failed downloading io/netty/netty-tcnative/2.0.19.Final/netty-tcnative-2.0.19.Final-$%7Bos.detected.classifier%7D.jar from https://jcenter.bintray.com/. Reason:
org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.netty:netty-tcnative:jar:${os.detected.classifier}:2.0.19.Final in jcenter (https://jcenter.bintray.com/)
Feb 03, 2020 3:18:59 PM org.jboss.shrinkwrap.resolver.impl.maven.logging.LogTransferListener transferFailed
WARNING: Failed downloading io/netty/netty-tcnative/2.0.19.Final/netty-tcnative-2.0.19.Final-$%7Bos.detected.classifier%7D.jar from https://repo.maven.apache.org/maven2/. Reason:
org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.netty:netty-tcnative:jar:${os.detected.classifier}:2.0.19.Final in central (https://repo.maven.apache.org/maven2)
Feb 03, 2020 3:18:59 PM org.jboss.shrinkwrap.resolver.impl.maven.logging.LogTransferListener transferFailed
WARNING: Failed downloading io/netty/netty-transport-native-unix-common/4.1.31.Final/netty-transport-native-unix-common-4.1.31.Final-$%7Bos.detected.name%7D-$%7Bos.detected.arch%7D.jar from https://repo.maven.apache.org/maven2/. Reason:
org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact io.netty:netty-transport-native-unix-common:jar:${os.detected.name}-${os.detected.arch}:4.1.31.Final in central (https://repo.maven.apache.org/maven2)
```

Reproducer is [here](https://github.com/galderz/jawa/blob/master/docker-java/src/docker_java.java).

**Comment by maxandersen** [comment]
shrinkwrap is for some reason grabbing these dependencies which doesn't even seem to be right to include so i'm reverting to use acegi. 

reported in https://issues.redhat.com/browse/SHRINKRES-302

**Comment by maxandersen** [comment]
fixed in  0.14
